### PR TITLE
Cleaned logging

### PR DESCRIPTION
**Pull request by hajothelen**
The SLF4J Logger was not correctly used. Now the Logger should be used in a proper way all over the project.

### CODE REVIEW

File: mjsip-phone/src/main/java/org/mjsip/phone/gui/StringList.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ protected void saveFile(File file) {
 			writeTo(new FileWriter(file));
 		}
 		catch (IOException e) {
-			LOG.error("Failed writing file \""+file+"\"", e);
+			LOG.error("Failed writing file ({})", file, e);
```
Comment (haumacher) [inline mjsip-phone/src/main/java/org/mjsip/phone/gui/StringList.java:71]: Is this really equivalent? The pattern has only one placeholder, and there are two arguments. Is this equivalent to the signature with message and exception?

**Review by haumacher — CHANGES_REQUESTED** [review]
Question regarding log signature - can the signature with message and exception changed to the pattern with arguments signature?

File: mjsip-phone/src/main/java/org/mjsip/phone/gui/StringList.java (hunk 1 above)
Comment (hajothelen) [reply mjsip-phone/src/main/java/org/mjsip/phone/gui/StringList.java:71]: Not 100%. 

The old entry produces `Failed writing file "filename"` the new one produces `Failed writing the file (filename)`

Comment (hajothelen) [reply mjsip-phone/src/main/java/org/mjsip/phone/gui/StringList.java:71]: And yes, both print an additional stack trace.

Comment (haumacher) [reply mjsip-phone/src/main/java/org/mjsip/phone/gui/StringList.java:71]: That's good to know, thank you!

**Comment by haumacher** [comment]
Thanks a lot for your contribution! That's a nice code improvement!

**Pull request merged**